suite-common/wallet-core/src/accounts/accountsThunks.ts

```diff
@@ -33,7 +33,7 @@ export const disableAccountsThunk = createThunk(
         const accounts = selectAccounts(getState());
         const enabledNetworks = selectEnabledNetworks(getState());
         // find disabled networks
-        const disabledNetworks = networksCompatibility
+        const disabledNetworks = networksCollection
             .filter(n => !enabledNetworks.includes(n.symbol) || n.isHidden)
             .map(n => n.symbol);
```

Comment:
:information_source: this is different! With `networksCompatibility`, the array of symbols is not unique, so e.g. `'btc'` is duplicated foreach account. Now with `networksCollection` it is unique.
BUt in this case it's ok, see L41, `disabledNetworks` is only used to test `.includes`